src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -37841,11 +37954,12 @@ class B : A
 }
 class C
 {
-    static void F((B?, B) x, (B, B?) y)
+    static void F((B?, B) x, (B, B?) y, (B, B) z)
     {
         (C, C?) c;
         c = x; // (ImplicitTuple)(ImplicitUserDefined)(ImplicitReference)
         c = y; // (ImplicitTuple)(ImplicitUserDefined)(ImplicitReference)
+        c = z; // (ImplicitTuple)(ImplicitUserDefined)(ImplicitReference)
```

Comment:
>ImplicitTuple [](start = 19, length = 13)

Please take a note of the scenario we discussed (with user-defined conversion introducing nullability) in the issue you opened covering explicit tuple conversions.

Reply:
Added scenario to [https://github.com/dotnet/roslyn/issues/32599](https://github.com/dotnet/roslyn/issues/32599).

---
In reply to: [249177167](https://github.com/dotnet/roslyn/pull/32612#discussion_r249177167) [](ancestors = 249177167)